WBooks/New Group/Library/Library_Cell/LibraryTableViewCell.swift

```diff
@@ -18,5 +18,20 @@ class LibraryTableViewCell: UITableViewCell, NibLoadable {
     
     @IBOutlet weak var frontBookImage: UIImageView!
     
-    @IBOutlet weak var bodyBook: UIView!
+    @IBOutlet weak var bodyBook: UIView! {
+        didSet {
+            bodyBook.layer.cornerRadius = 10
+        }
+    }
+}
+
+internal extension LibraryTableViewCell {
+    
+    func bind(viewModel: BookViewModel){
+        titleLabel.text = viewModel.title
+        authorLabel.text = viewModel.author
+        viewModel.downloadImage(closure: {
```

Comment:
dont forget to add `[unowned self] in ` at the start of the closure if you use self, it avoids retaining the reference to self. The rest is ok, but you can also write it like this:
```
viewModel.downloadImage() { [unowned self] in
   self.frontBookImage.image = $0
}

```